controllers/admin/AdminStatsController.php

```diff
@@ -199,37 +199,80 @@ public static function getTotalSales($date_from, $date_to, $granularity = false)
     {
         if ($granularity == 'day') {
             $sales = array();
-            $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->ExecuteS('
-			SELECT LEFT(`date_upd`, 10) as date, SUM(total_paid_tax_excl / o.conversion_rate) as sales
-			FROM `'._DB_PREFIX_.'orders` o
-			LEFT JOIN `'._DB_PREFIX_.'order_state` os ON o.current_state = os.id_order_state
-			WHERE `date_upd` BETWEEN "'.pSQL($date_from).' 00:00:00" AND "'.pSQL($date_to).' 23:59:59" AND os.logable = 1 AND os.paid = 1
-			'.Shop::addSqlRestriction(false, 'o').'
-			GROUP BY LEFT(`date_upd`, 10)');
-            foreach ($result as $row) {
+            $resultOrders = Db::getInstance(_PS_USE_SQL_SLAVE_)->ExecuteS('
+                SELECT LEFT(o.date_upd, 10) as date, SUM((o.total_paid_tax_excl - coalesce(osl.total_products_tax_excl,0)) / o.conversion_rate) as sales
+                FROM `' . _DB_PREFIX_ . 'orders` o
+                LEFT JOIN `' . _DB_PREFIX_ . 'order_state` os ON o.current_state = os.id_order_state
+                LEFT JOIN `' . _DB_PREFIX_ . 'order_slip` osl ON o.id_order = osl.id_order
+                WHERE o.date_upd BETWEEN "' . pSQL($date_from) . ' 00:00:00" AND "' . pSQL($date_to) . ' 23:59:59" AND os.logable = 1 AND os.paid = 1
```

Comment:
@eternoendless I have to rebase this one after merging yours [#8335](https://github.com/PrestaShop/PrestaShop/pull/8335)

Reply:
Done